pkg/client/client_venconn.go

```diff
@@ -162,20 +162,14 @@ func (c *VenConnClient) PostDataReadingsWithOptions(ctx context.Context, reading
 	// The path parameter "no" is a dummy parameter to make the Venafi Cloud
 	// backend happy. This parameter, named `uploaderID` in the backend, is not
 	// actually used by the backend.
-	req, err := http.NewRequestWithContext(ctx, http.MethodPost, fullURL(token.BaseURL, "/v1/tlspk/upload/clusterdata/no"), encodedBody)
+	req, err := http.NewRequestWithContext(ctx, http.MethodPost, fullURL(details.VCP.URL, "/v1/tlspk/upload/clusterdata/no"), encodedBody)
 	if err != nil {
 		return err
 	}
 
 	req.Header.Set("Content-Type", "application/json")
 	req.Header.Set("User-Agent", fmt.Sprintf("venafi-kubernetes-agent/%s", version.PreflightVersion))
-
-	if token.VCPAccessToken != "" {
-		req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", token.VCPAccessToken))
-	}
-	if token.VCPAPIKey != "" {
-		req.Header.Set("tppl-api-key", token.VCPAPIKey)
-	}
```

Comment:
question: without knowing why this changed it looks scary. Is the VCP API key deprecated or something?